devonfw-ide/index.asciidoc

```diff
@@ -0,0 +1,14 @@
+= Devonfw IDE Setup
+====
+Devon offers complete IDE solution integrated with a lot​ of integrated tooling and initial settings​. 
+It massively speeds up development, reduces risks and helps deliver better results.
+
+More information about DEVONFW IDE on https://devonfw.com/website/pages/docs/devonfw-ide-introduction.asciidoc.html
+====
+
+Below steps will guide in the IDE Setup
+[step]
+--
+installDevonfwIde(["java","mvn"])
+--

```

Comment:
After the step you could add some additional text and say that in this case java and maven were installed inside the IDE. And that the user can change the tools by changing the devon.properties file.

Reply:
Yes, it was in a draft version. I had planned to add this. Taken care now.